### ISSUE TITLE
Mistake in your code! - Batch Norm in Encoder

### ISSUE THREAD

**Issue by soulslicer** [JiaRenChang/PSMNet#183]
I noted that in your model, everywhere that BatchNorm is used (track_running_stats) is False!

But in this one part here, it is True.

https://github.com/JiaRenChang/PSMNet/blob/master/models/submodule.py#L108

Why is this the case??

**Comment by JiaRenChang** [comment]
@soulslicer 
In training, track_running_stats of all BN should be True.
